Assets/LeapMotion/Core/Scripts/DataStructures/ReadonlyList.cs

```diff
@@ -40,8 +47,20 @@ public struct ReadonlyList<T> {
       return _list.GetEnumerator();
     }
 
+    IEnumerator<T> IEnumerable<T>.GetEnumerator() {
```

Comment:
If you use the List<T>.Enumerator you will be able to foreach a ReadonlyList without allocating garbage.

Reply:
will fix

Reply:
hm OK. I wonder why I marked ReadonlyList<T> as IEnumerable since we know that allocates. /shrug